server/form-pages/apply/risk-and-need-factors/oasys-import/optionalOasysSections.ts

```diff
@@ -7,8 +7,11 @@ import { DataServices } from '../../../../@types/ui'
 import { sentenceCase } from '../../../../utils/utils'
 
 interface Response {
-  needsLinkedToReoffending: Array<string> | string | Array<OASysSection>
-  otherNeeds: Array<string> | string | Array<OASysSection>
+  /** @nullable */
+  needsLinkedToReoffending: Array<string> | string | Array<OASysSection> | null
+  /** @nullable */
+
+  otherNeeds: Array<string> | string | Array<OASysSection> | null
```

Comment:
That empty line at 13 shouldn't be there.  I think it's stopped the schema generator from making either of those fields nullable